docs/releases.rst

```diff
@@ -1,6 +1,95 @@
 Release notes
 =============
 
+Mailu 1.6 - 2019-01-18
+----------------------
+
+Its been more than a year since the release of 1.5! And what a year it has been...
+More then 800 commits are done since 1.5, containing thousands of additions.
+We had the honor of welcoming more and more contributors and we actually established
+a dedicated team of trusted contributors.
+
+With new review guidelines we now allow the project to grow without dependence
+on any single person. And thus merging pull requests at much shorter time.
+On top of that we finally got around to creating a simple test suite on TravisCI,
+which is doing some e-mail sending and receiving. This greatly helps the reviewing process.
+
+Full a complete overview of changes, see our `changelog`_.
+Here we'll try to give you the highlights.
+
+.. _`changelog`: https://github.com/Mailu/Mailu/blob/master/CHANGELOG.md
+
+New functionality
+`````````````````
+
+We offer a lot new functions in the user experience. Some of the highlights would be quota
+support from the admin interface, optional user sign up with recaptcha, auto-reply start date,
+and a client setup page.
+
+Mailu now also offers a `setup utility`_.
+This utility helps users to generate a `docker-compose.yml` and `mailu.env` through guided steps.
+
+Documentation
+`````````````
+
+Quite some efforts were done in expanding the documentation of Mailu.
+We've added support for :ref:`kubernetes`, `Docker Swarm`_ and a :ref:`faq` section.
+There is now also a section on running the Mailu web interfaces behind :ref:`traefik_proxy`.
+
+We now also Dockerized the documentation, allowing for easy local running and versions
+management on our web server.
+
+.. _`Docker Swarm`: https://github.com/Mailu/Mailu/blob/master/docs/swarm/master/README.md
+
+Back-end
+````````
+
+Lots and lots of hours went in to the back-end. Work on numerous bugs,
+increased the general performance and allowing for better maintainability.
+
+We've reworked the complete interface with the database. All queries are now done
+through the Admin container, with that being the single point of contact with the
+database. Now we also support the usage of MySQL and PostgreSQL databases and Mailu
+comes with its own PostgreSQL image! This allows for Mailu to be used in larger scaled
+operations.
+
+Main software versions
+``````````````````````
+- Alpine 3.8.1
```

Comment:
I think we are currently on `Alpine 3.8.2`

Reply:
Yes, you are right. It seems my laptop had an old base image cached in Docker. On my servers its 3.8.2 (they pull from docker hub)